api/services/elastigroup/aws/schemas/elastigroup-scale-down.yaml

```diff
@@ -0,0 +1,120 @@
+    example: "percent"
+  threshold:
+    type: integer
+    description: |
+      The value against which the specified statistic is compared
+    example: 90
+  action:
+    type: object
+    example: "adjustment"
+    description: |
+      The action to take when scale up is needed.
+      A mandatory field, that should be filled with one of the following:
+      "adjustment", "percentageAdjustment", "setMaxTarget", "setMinTarget", "updateCapacity".
+    properties:
+      type:
+        type: string
+        description: |
+          The type of the action to take when scale up is needed.
+        enum: [ adjustment , updateCapacity setMinTarget , percentageAdjustment ]
+        example: "updateCapacity"
+      adjustment:
+        type: integer
+        description: |
+          The number / percentage associated with the specified adjustment type.
+          Required if using `adjustment` or `percentageAdjustment` as action type
+        example: 1
+      minTargetCapacity:
+        type: integer
+        description: |
+          The number with the target capacity needed.
+          Required if using `setMinTarget` as action type
+        example: 1
+      target:
+        type: integer
+        description: |
+          The desired number of instances. Required if using `updateCapacity` as action type and neither `minimum` nor `maximum` are not defined.
+        example: 1
+      minimum:
+        type: integer
+        description: |
+          The lower limit number of instances that you can scale down to. Optional, required if using `updateCapacity` as action type and neither `target` nor `maximum` are not defined
+        example: 0
+      maximum:
+        type: integer
+        description: |
+          The upper limit number of instances that you can scale up to. Optional, required if using `updateCapacity` as action type and neither `target` nor `minimum` are not defined
+        example: 5
+  namespace:
+    type: string
+    description: |
+      The namespace for the alarm's associated metric.
+    default: "AWS/EC2"
+  dimensions:
+    type: array
+    description: |
+      The dimensions for the alarm's associated metric.
+      If the user mentioned name as instanceId, there is NO value
+    items:
+      type: object
+      properties:
+        name:
+          type: string
+          example: "InstanceId"
+  period:
+    type: integer
+    description: |
+      The period in seconds over which the statistic is applied.
+    example: 300
+  evaluationPeriods:
+    type: integer
+    description: |
+      The number of periods over which data is compared to the specified threshold.
+    example: 1
+  cooldown:
+    type: integer
+    description: |
+      The amount of time, in seconds, after a scaling activity completes before any further trigger-related scaling activities can start.
```

Comment:
"The amount of time, in seconds, after a scaling activity completes and before..."